Update README.md (#48)
r-devel/r-dev-env: README.md

### Codespaces

Click on the 'Open in GitHub Codespaces' button and then click the green 'Create Codespace' button.

> > You will see the message "Codespace usage for this repository is paid for by ", with your username. Don't panic!

Reviewer: ```suggestion
> You will see the message "Codespace usage for this repository is paid for by ...", with your username. Don't panic!
```

> > **Note : The Github Codespaces is offered 120 core hours of free usage per month for every Github User. That means one can use codespace for 120hrs of free and its get reset every month.

Reviewer: ```suggestion
> **Note : Github Codespaces offers 120 core hours of free usage per month for every Github user.
```

> The free hours also differ according to the number of cores you are using to run your codespaces.**

Reviewer: ```suggestion
So the actual number of free hours is 120 divided by the number of cores you are using to run your codespaces.**
```

>

> > **Here for the R-dev-env codespaces we have set the codespace usage to 4 cores which leads to 30hrs of free usage per month. And it can also be changed according to your preference.**

Reviewer: ```suggestion
> **Here for the R-dev-env codespace we have set the codespace usage to 4 cores which leads to 30hrs of free usage per month. And it can also be changed according to your preference.**
```

> > For more details about codespaces billing, you can read this [Codespaces Billing Docs](https://github.com/features/codespaces). And also calculate your Github services usage using this [Github Services Pricing Calculator](https://github.com/pricing/calculator). Moreover you can check your usage allowance under "Codespaces" on https://github.com/settings/billing.

Reviewer: ```suggestion
> For more details about codespaces billing, see the [Codespaces Billing Docs](https://github.com/features/codespaces). You can calculate your GitHub services usage with the [GitHub Services Pricing Calculator](https://github.com/pricing/calculator) and check your usage allowance under "Codespaces" on https://github.com/settings/billing.
```

Reviewer: Can you update the last sentence as suggested, then this PR should be ready to merge.
